.github/actions/gh-api-update-check/action.yml

```diff
@@ -39,23 +39,43 @@ inputs:
     description: "The text to display next to the check"
     default: ""
     required: false
-  context:
-    description: "The name of the status check"
+  title:
+    description: "The title of the status check"
     required: true
-  state:
-    description: "The state of the check. Can be one of: error, failure, pending, success"
+  status:
+    description: "The status of the check. Can be one of: queued, in_progress, completed, waiting, requested, pending"
     required: true
+  conclusion:
+    description: "Required if status is 'completed'. Can be one of: action_required, cancelled, failure, neutral, success, skipped, stale, timed_out"
+    required: false
 
 runs:
   using: "composite"
   steps:
-    - name: Update Check
+    - if: inputs.conclusion == ''
```

Comment:
Pardon me, what is the purpose of this step?

Reply:
The checks API has status and conclusion. The conclusion is only given if the status is "completed". 

If we want to set a check to pending (as we've discussed in other PRs), we would send status=pending with no conclusion. 

Because of the two forms of API call needed (one with conclusion, one without), we needed two steps with different arguments.

Reply:
@mumrah, thanks for sharing. Although this step is not currently in use, it is intended to serve the needs of [KAFKA-17607](https://issues.apache.org/jira/browse/KAFKA-17607), right?

Reply:
Maybe you meant https://issues.apache.org/jira/browse/KAFKA-17572 (clearing a status when re-running a job)?

Reply:
oh, you are right 😀